SQL/Archive/18.0/2018-01-18-remove-radiological-review-menu.sql

```diff
@@ -0,0 +1,11 @@
+WARNINGS;
```

Comment:
shouldn't this go to the 19.0-dev branch?

Reply:
if so, i'll have to close this PR and make another. lmk